config/charger/animation.txt

```diff
@@ -0,0 +1,81 @@
+## Google Pixel animation
+
+# animation: num_cycles, first_frame_repeats, animation_file
+animation: 5 1 charger/battery_scale
+
+# percent_display: text_percent_pos_x, text_percent_pos_y, color_r, color_g, color_b, color_a, font_file
+percent_display: c c 255 255 255 255 charger/main_font
```

Comment:
As we can at least easily change the color here, how does it look like if we use:

`percent_display: c c 22 124 128 255 charger/main_font`

Which should be the LineageOS color?

Reply:
No, it's some sort of yellow now

Reply:
isn't that RGBA? Hmm

Reply:
Ah, I think it is rgba(22, 124, 128, 1)

Reply:
That just makes the percentage disappear ..

Reply:
Works now with 124 128 22 255

Reply:
It should be rgba but for some reason it only works with gbra

Reply:
Maybe related to https://github.com/whatawurst/android_device_samsung_exynos9820-common/blob/lineage-17.1/BoardConfigPlatform.mk#L195 ?